### DISCUSSION TITLE
Upload Files instead of photos for verification

### DISCUSSION THREAD

**Discussion by W-Garlington** [community/community#129013] [GitHub Education]
### Hi everyone,

I know writing this wont fix anything but it is very lame that you can't upload documents to verify student status. I saw an answer in some other thread about "wanting students to be on campus" but even if I was on campus, I'd be hard pressed to find a document that was mailed to me rather than sent electronically. It's hilarious that the only work around I can think of is printing out my documents for the sole purpose of taking a photo of them. Also L + ratio to the PR guy who eventually responds to this.
# TLDR: Do better.

**Comment by DeviantNinja** [comment]
I agree on this one, most communication these days is done by email, I have no paper documentation. Just emails stating my enrollment, I have asked for a letter from the university to prove my enrollment but even that's going to be emailed to me.

Secondly When trying to take a photo it will only work with my front facing camera, and doesn't load my back camera regardless of what choice i make. It seems like a great benefit to students tot get using Git and the other education material early.
